### ISSUE TITLE
Unicode support

### ISSUE THREAD

**Issue by lemmy** [tlaplus/tlaplus#64]
@pron added unicode support to the Toolbox. The patch awaits merging in the [pron-unicode branch](https://github.com/tlaplus/tlaplus/tree/pron-unicode).

**Comment by lemmy** [comment]
Done with commit d86dcb4542049c9c5c59f366466c90b4d0fdd459

**Comment by lemmy** [comment]
The patch is not ready yet and has been reverted in the master branch.

**Comment by sophistifunk** [comment]
Wouldn't the easiest solution to this being extending something like Fira Code with a few TLA-specific ligatures?

**Comment by pron** [comment]
No, because then the ligature for `\sqsubseteq` would need to be 11 characters wide.
